code/TemplateStudioForWinUICs/Templates/Proj/Default/Param_ProjectName/App.xaml.cs

```diff
@@ -1,23 +1,37 @@
 ﻿using Microsoft.UI.Xaml;
 using Param_RootNamespace.Helpers;
 
-// To learn more about WinUI3, see: https://docs.microsoft.com/windows/apps/winui/winui3/.
 namespace Param_RootNamespace;
 
+// To learn more about WinUI 3, see https://docs.microsoft.com/windows/apps/winui/winui3/.
 public partial class App : Application
 {
-    public static Window MainWindow { get; set; } = new Window() { Title = "AppDisplayName".GetLocalized() };
+    public static WindowEx MainWindow { get; } = InitializeMainWindow();
+
+    private static WindowEx InitializeMainWindow()
+    {
+        return new WindowEx()
+        {
+            Backdrop = new MicaSystemBackdrop(),
+            Content = null,
+            MinHeight = 500,
+            MinWidth = 500,
+            PersistenceId = "MainWindow",
+            Title = "AppDisplayName".GetLocalized()
```

Comment:
It'll default to a white background, even in dark mode, so it looks pretty terrible. I've generally used `#333` for dark mode and `#eee` for light mode, as it seems to match mica pretty closely.

Reply:
Also see https://github.com/microsoft/WindowsAppSDK/issues/2637

Reply:
> Background="{ThemeResource SolidBackgroundFillColorBaseBrush}" is what we were using.

Yeah that works on the page element, but that would block the view of the backdrop that is applied to the Window.

Reply:
Yes that's why we removed it when adding Mica. I'll test on a Win10 VM, see how it behaves, and adjust Fallback appropriately.

Reply:
No there's no way to get that without a direct reference to the application resources - which you can't get in a class library (with UWP you could do it via Application.Current but that's not available in WinUI3). The Mica class has no way to get a reference to the root window element and its resources. Hence why I logged issue 7305.

Reply:
We can hardcode the values in the library. I filed https://github.com/microsoft/TemplateStudio/issues/4563.

Reply:
We could - but that doesn't feel right. The default values match the default values of WinAppSDK's Mica controller.

Reply:
The default behavior is broken though when the fallbacks are used right? As you pointed out, without setting fallbacks the current default behavior is broken in both themes when Mica isn't available. I'd rather the library default to functional behavior that uses the recommended colors from the WinUI designers.

Reply:
IMHO this makes sense to set in the application that declares the background, rather than as a default - I don't see it much different than your example above where you bind the themeresource to the background brush of the root element.
It also makes it more discoverable for developers where to change it if they do decide on changing styling from default in template studio.

Reply:
I'd strongly recommend and prefer the library defaulting to functional behavior over broken behavior with the bonus that it aligns with WinUI design guidelines out-of-the-box. The current Fallback properties are not very discoverable and there are no hints that they are required to achieve functional behavior downlevel. Not everyone uses Template Studio so setting it in Template Studio doesn't help WinUIEx users who aren't using Template Studio to achieve the desired behavior.

Reply:
How about doing this in the app?
```cs
var lightresources = Application.Current.Resources.MergedDictionaries.OfType<Microsoft.UI.Xaml.ResourceDictionary>().FirstOrDefault()?.ThemeDictionaries["Light"] as ResourceDictionary;
if (lightresources?["SolidBackgroundFillColorBase"] is Windows.UI.Color color)
    micaBackdrop.LightFallbackColor = color;
```

Reply:
We'll do what we need do in the meantime but it doesn't really change my argument.

Reply:
ok I'll consider it some more.

Reply:
I'm finding SolidBackgroundFillColorBase is `#F3F3F3` in light mode and `#202020` in darkmode. Where do you get your values from?